### ISSUE TITLE
Support plotting histograms

### ISSUE THREAD

**Issue by EwoutH** [Corvince/mesa-interactive#8]
Being able to plot histograms could be very useful, for example to plot a distribution of wealth.

**Comment by Corvince** [comment]
Yes that should be easy to implement. Additionally we should have a generic Interface where users just provide a Altair spec with some constraints on the data and mesa interactive just handles updating the data
